LICENSE-binary

```diff
@@ -328,6 +328,13 @@ Eclipse Distribution License - v 1.0
 ------------------------------------
 jakarta.activation:jakarta.activation-api
 jakarta.xml.bind:jakarta.xml.bind-api
+org.eclipse.collections:eclipse-collections-api
+org.eclipse.collections:eclipse-collections
+
+Eclipse Public License (EPL) 1.0
+--------------------------------
+org.eclipse.collections:eclipse-collections-api
```

Comment:
for dual Licensed dependency, we only need to choose one. for this case, let's choose EDL(listed in category A) https://apache.org/legal/resolved.html